Doc/library/asyncio-sync.rst

```diff
@@ -216,7 +216,7 @@ Condition
 
    .. method:: notify(n=1)
 
-      Wake up at most *n* tasks (1 by default) waiting on this
+      Wake up *n* tasks (1 by default) waiting on this
       condition.  The method is no-op if no tasks are waiting.
```

Comment:
Because there *might* be more than one woken up.  Condition variables (in general, in computing) make no guarantees that a task is _not_ woken up when not explicitly requested.  Only that one is _definitely_ woken up when asked to do so.  It is by design allowed to err to the sign of caution.  Better to have to many wakeups than to few, since the latter can cause deadlocks.  This is also the reason why any task awoken _must_ check the state, to see if it should do anything or just go back to sleep.

Reply:
Thank you.  I guess it bugged me that the "at most" sounded like a guarantee.